src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/BasicModelDefTest.java

```diff
@@ -150,5 +150,13 @@ public void globalLibrarySuccess() throws Exception {
 
         j.assertLogContains("[nothing here]", b);
         j.assertLogContains("map call(1,2)", b);
+        /* TODO called the wrong way: closure1([a:1])
```

Comment:
fwiw, I tried this as just a simple job with a function in the Pipeline -
```groovy
closure1(a: 1) { echo "Yo" }

def closure1(a, Closure body) { echo "closure1($a)"; body() }
```

...and it did the same thing.